tests/jest/config/webpack/loaders/pretendExample2.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-absolute-path */
+import React from 'react';
+import classNames from 'classnames/bind';
+// eslint-disable-next-line import/no-unresolved
+import styles from './pretendExampless.scss';
```

Comment:
```suggestion
import styles from './pretendExamples.scss';
```

Reply:
Originally the typo was on purpose to simulate an unresolvable path, but the test and associated examples have been removed: [b3dc614](https://github.com/cerner/terra-dev-site/pull/281/commits/b3dc614f73a306fd42797d018815ac6977eb4387).